spyderlib/spyder.py

```diff
@@ -589,33 +589,44 @@ def setup(self):
                                             context=Qt.WidgetShortcut)
             self.register_shortcut(self.replace_action, "Editor",
                                    "Replace text")
+
             def create_edit_action(text, tr_text, icon_name):
                 textseq = text.split(' ')
                 method_name = textseq[0].lower()+"".join(textseq[1:])
-                return create_action(self, tr_text,
-                                     shortcut=keybinding(text.replace(' ', '')),

```

Comment:
Why did you remove this line? I think without it shortcuts are not displayed next to menu entries.